table/block_based/block_based_table_factory.cc

```diff
@@ -234,6 +234,7 @@ static std::unordered_map<std::string, OptionTypeInfo>
         /* currently not supported
           std::shared_ptr<Cache> block_cache = nullptr;
           std::shared_ptr<Cache> block_cache_compressed = nullptr;
+          CacheUsageOptions cache_usage_options;
```

Comment:
NOTE:
IIUC, this indicates that we do not support dynamic update to memory_tracking_override and serialization/de-serialization. I am not quite sure about the latter so haven't added the note to API, which I should after confirming on this.